tests/conftest.py

```diff
@@ -84,18 +84,24 @@ def finalizer():
 
 
 @pytest.fixture()
-def block_pool_factory(request):
+def ceph_pool_factory(request):
     """
     Create a Ceph block pool factory.
     Calling this fixture creates new block pool instance.
     """
     instances = []
 
-    def factory():
-        cbp_obj = helpers.create_ceph_block_pool()
-        assert cbp_obj, "Failed to create block pool"
-        instances.append(cbp_obj)
-        return cbp_obj
+    def factory(interface=constants.CEPHBLOCKPOOL):
+        if interface == constants.CEPHBLOCKPOOL:
+            ceph_pool_obj = helpers.create_ceph_block_pool()
+        elif interface == constants.CEPHFILESYSTEM:
+            cfs = ocp.OCP(kind=constants.CEPHFILESYSTEM).get(
```

Comment:
I think you need to initialize OCP with openshift-storage namespace

Reply:
Not mandatory as the ceph pool is namespace-less